RtspServer/RtspServer.cs

```diff
@@ -477,14 +471,7 @@ public IEnumerable<Media.Rtsp.Server.IMedia> MediaStreams
             [System.Runtime.CompilerServices.MethodImpl(System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]
             get
             {
-                try
-                {
-                    return m_MediaStreams.Values.ToList();
-                }
-                catch
-                {
-                    throw;
-                }
+                return m_MediaStreams.Values.ToList();
```

Comment:
Rather than allocate here, because you are using ConcurrentDictionary I think you can just use Values here and skip the ToList.